vpr/src/place/initial_noc_placement.cpp

```diff
@@ -291,4 +295,39 @@ void initial_noc_placement(const t_noc_opts& noc_opts, const t_placer_opts& plac
                         "At least one cycle was found in NoC channel dependency graph. This may cause a deadlock "
                         "when packets wait on each other in a cycle.\n");
     }
+
+//    const auto& placement_ctx = g_vpr_ctx.placement();
```

Comment:
Need to have a comment saying what this code is for (can point at a certain block comment in some file too).